docs/admin/install/install.rst

```diff
@@ -76,9 +83,9 @@ Users of SecureDrop Workstation must enter their username, passphrase and two-fa
 
 In order to set up KeePassXC for easy use:
 
-- Add KeePassXC to the application menu by selecting it from the list of available apps in **Q > Domain: vault > vault: Qube Settings > Applications** and pressing the button labeled **>** (do not press the button labeled **>>**, which will add *all* applications to the menu).
+- Add KeePassXC to the application menu by selecting it from the list of available apps in **Q > Apps > vault > Settings > Applications** and pressing the button labeled **>** (do not press the button labeled **>>**, which will add *all* applications to the menu).
```

Comment:
This is a pre-existing issue, but "Applications" is part of a separate menu, and I think it's misleading to make it think it's part of the same Q menu.

(Not asking for a fix, just flagging for a future potential improvement)